### ISSUE TITLE
ipython breaks on Python-3.13.0b1 when a 'tempfilepager' is not defined 

### ISSUE THREAD

**Issue by stonebig** [python/cpython#119185]
# Bug report

### Bug description:

on a windows Python-3.13.0b1 non-english

test:
```python
Python 3.13.0b1 (tags/v3.13.0b1:2268289, May  8 2024, 12:20:07) [MSC v.1938 64 bit (AMD64)]
Type 'copyright', 'credits' or 'license' for more information
IPython 8.24.0 -- An enhanced Interactive Python. Type '?' for help.

In [1]: help(len)
```

result:
```python
Unexpected exception formatting exception. Falling back to standard exception
Traceback (most recent call last):
  File "......\python-3.13.0b1.amd64\Lib\site-packages\IPython\core\interactiveshell.py", line 3577, in run_code
    exec(code_obj, self.user_global_ns, self.user_ns)
    ~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<ipython-input-1-1dda769017ba>", line 1, in <module>
    help(len)
    ~~~~^^^^^
  File "<frozen _sitebuiltins>", line 103, in __call__
  File "......\python-3.13.0b1.amd64\Lib\pydoc.py", line 1983, in __call__
    self.help(request)
    ~~~~~~~~~^^^^^^^^^
  File "......\python-3.13.0b1.amd64\Lib\pydoc.py", line 2044, in help
    else: doc(request, 'Help on %s:', output=self._output, is_cli=is_cli)
          ~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "......\python-3.13.0b1.amd64\Lib\pydoc.py", line 1757, in doc
    pager(render_doc(thing, title, forceload), f'Help on {what!s}')
    ~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "......\python-3.13.0b1.amd64\Lib\pydoc.py", line 1656, in pager
    pager(text, title)
    ~~~~~^^^^^^^^^^^^^
  File "......\python-3.13.0b1.amd64\Lib\_pyrepl\pager.py", line 38, in <lambda>
    return lambda text, title='': tempfilepager(plain(text), 'more <')
                                  ^^^^^^^^^^^^^
NameError: name 'tempfilepager' is not defined. Did you mean: 'tempfile_pager'?
```

### CPython versions tested on:

3.13

### Operating systems tested on:

Windows

### Linked PRs
* gh-118881
* gh-119211

**Comment by stonebig** [comment]
the fingerprint says:
![image](https://github.com/python/cpython/assets/4312421/f87afe7a-f5d3-411e-9a20-e8a43f6ab99a)

**Comment by graingert** [comment]
This is a typo it should be tempfile_pager

**Comment by Eclips4** [comment]
We have already a PR for fix that: #118881

**Comment by vstinner** [comment]
Fixed by https://github.com/python/cpython/commit/05e1dce76d7669e90ab73e7e682360d83b8a0d02
